### PR TITLE
podlet 0.3.0 (new formula)

### PR DESCRIPTION
**Pull request by cprecioso**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

podlet converts Docker compose files to the quadlet format from podman.
It has recently [been adopted into the containers project](https://github.com/containers/podlet) (same as podman and buildah).

### CODE REVIEW

File: Formula/p/podlet.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Podlet < Formula
+  desc "Generate podman quadlet files from a podman command or compose file"
+  homepage "https://github.com/containers/podlet"
+  url "https://github.com/containers/podlet/archive/refs/tags/v0.2.4.tar.gz"
```
Comment (chenrui333) [inline Formula/p/podlet.rb:4]: should be 0.3.0? can you raise an issue to ask about it?

- https://github.com/containers/podlet/pull/77

File: Formula/p/podlet.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  sha256 "3774e49ee0826221772c06d37478c1cd421d8cbf00cda6b16605df4b9f8b2292"
+  license "MPL-2.0"
+  head "https://github.com/containers/podlet.git", branch: "main"
+  livecheck do
+    url :head
+  end
```
Comment (chenrui333) [inline Formula/p/podlet.rb:10]:
```suggestion
```

Comment (cprecioso) [reply Formula/p/podlet.rb:10]: Is the livecheck not needed anymore?

Comment (SMillerDev) [reply Formula/p/podlet.rb:10]: The head url is the same as the download url. No need to specify a livecheck with that

**Comment by chenrui333** [comment]
looks like upstream gonna make a 0.3.0 release soon, let's hold a bit for it.

**Review by cprecioso — COMMENTED** [review]
@chenrui333 updated to 0.3.0

**Review by SMillerDev — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@cprecioso, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**